molgenis/molgenis-emx2
# fix: upload of same data model should not give errors. closes #3655

closes #3655

What are the main changes you did:
- added test that failed when importing same model twice
- issue was that if/then didn't scope for inherited tables

how to test:
- see #3655 
- or otherwise; download a data model having inheritance and then upload again

done:
